frontend/apps/crates/utils/js/mixpanel.js

```diff
@@ -0,0 +1,6 @@
+import Mixpanel from 'mixpanel-browser';
+// Weird hacky solution. But I can't get wasm_bindgen to define a static_method_of Mixpanel without it
+// also attempting to create an import like `import {Mixpanel} from '...'`.
+//
+// So this just exports Mixpanel in a way that makes everyone happy.
+export {Mixpanel}
```

Comment:
Does this help you? https://rustwasm.github.io/docs/wasm-bindgen/print.html#js_name--blah

> The js_name attribute can also be used in situations where a JavaScript module uses export default. In this case, setting the js_name attribute to "default" on the type declaration, and the [js_class attribute](https://rustwasm.github.io/docs/wasm-bindgen/js_class.html) to "default" on any methods on the exported object will generate the correct imports.

Reply:
Nope. I've tried those. mixpanel's JS lib doesn't export a class like normal. It uses a singleton. So declaring the type and annotating with `js_name = default` and then the functions with `js_class = default` breaks the build. We need to use static_method_of so that we can call `mixpanel::<whatever>`, but that doesn't work with default. Hence this solution